tracing-subscriber/src/fmt/fmt_layer.rs

```diff
@@ -0,0 +1,405 @@
+use crate::{
+    field::RecordFields,
+    fmt::{format, FormatEvent, FormatFields, MakeWriter},
+    layer::{Context, Layer},
+    registry::{LookupMetadata, LookupSpan, SpanRef},
+};
+use std::{cell::RefCell, fmt, io, marker::PhantomData};
+use tracing_core::{
+    span::{Attributes, Id, Record},
+    Event, Subscriber,
+};
+
+/// A [`Layer`] that logs formatted representations of `tracing` events.
+///
+/// [`Layer`]: ../trait.Layer.html
+#[derive(Debug)]
+pub struct FmtLayer<
+    S,
+    N = format::DefaultFields,
+    E = format::Format<format::Full>,
+    W = fn() -> io::Stdout,
+> {
+    make_writer: W,
+    fmt_fields: N,
+    fmt_event: E,
+    _inner: PhantomData<S>,
+}
+
+/// A builder for `FmtLayer` that logs formatted representations of `tracing`
```

Comment:
Can we linkify `FmtLayer`?
```suggestion
/// A builder for [`FmtLayer`](struct.FmtLayer.html) that logs formatted representations of `tracing`
```